corehq/apps/importer/util.py

```diff
@@ -121,6 +121,17 @@ def __init__(self, file_path, column_headers):
         except Exception:
             self.has_errors = True
 
+    def _col_values(self, sheet, index):
+        return [self._fmt_value(cell) for cell in sheet.col(index)]
+
+    def _row_values(self, sheet, index):
+        return [self._fmt_value(cell) for cell in sheet.row(index)]
+
+    def _fmt_value(self, cell):
+        if cell.ctype == XL_CELL_NUMBER and int(cell.value) == cell.value:
```

Comment:
i'm reading it as there's a cell that is a `XL_CELL_NUMBER` type and that the int version is equal to the regular version. doesn't that mean the cell is already an integer? or is this for converting decimals?